tokio/src/runtime/builder.rs

```diff
@@ -489,6 +496,31 @@ cfg_time! {
     }
 }
 
+cfg_test_util! {
+    impl Builder {
+        /// Controls if the runtime's clock starts paused or advancing.
+        ///
+        /// Pausing time requires the current-thread runtime; construction of
+        /// the runtime will panic otherwise.
```

Comment:
Where is this panic?

Reply:
The call to `clock.pause` in `Clock::new` will panic when running on a multi threaded runtime.